src/app/code/community/Omise/Gateway/Model/Api/Charge.php

```diff
@@ -161,9 +161,7 @@ public function isUnauthorized()
      */
     public function isPaid()
     {
-        $paid = isset($this->paid) ? $this->paid : $this->captured;
```

Comment:
Change is required for the change to new API version, which in turn is required to implement instalmnet payments, so is completely relevant to PR as stuff would break without it

Reply:
This line has already prevented by either use `paid` if it's presented or `captured` if it isn't. 
I believe you can update API version without making a change on this line (maybe I'm wrong thou).
But anyway, that was before I saw https://github.com/omise/omise-magento/pull/169/files#diff-5560d36b525b25eb4fe67c06b341a30bR81

Reply:
I had to make the change precisely because of the API change